Alignment/CommonAlignmentProducer/bin/aligncond_split_iov.cc

```diff
@@ -1,185 +0,0 @@
-#include "CondCore/DBCommon/interface/DbSession.h"
```

Comment:
@mmusich we could provide SOME replacement but I have no idea what this thing was meant for.